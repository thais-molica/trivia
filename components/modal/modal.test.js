import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Modal from "./index";

it("Modal renders correctly", () => {
  const tree = renderer.create(<Modal/>).toJSON();
  expect(tree).toMatchSnapshot();
});
