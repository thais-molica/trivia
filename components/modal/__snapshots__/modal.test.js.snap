// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal renders correctly 1`] = `
Array [
  <span
    className="jsx-3314422632"
  />,
  <div
    className="jsx-3314422632 jsx-654452721"
  >
    <em
      className="jsx-3314422632"
    >
      <img
        alt="erroe"
        className="jsx-3314422632"
        src={[Function]}
      />
    </em>
    <p
      className="jsx-3314422632"
    >
      Wrong answer!
    </p>
    <button
      className="jsx-1173346177"
      onClick={[Function]}
    >
      Next 
      <img
        alt="Next"
        className="jsx-3314422632"
        src={[Function]}
      />
    </button>
  </div>,
]
`;
